tests/AssetManagerTest/View/Helper/AssetTest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+namespace AssetManagerTest\View\Helper;
+
+use PHPUnit_Framework_TestCase as TestCase;
+use AssetManager\View\Helper\Asset;
+use Zend\ServiceManager\ServiceManager;
+
+class AssetTest extends TestCase
+{
+    public function testInvoke()
+    {
+        /* TODO need help
```

Comment:
You don't need the `ServiceManager` if you do proper injection and mock all the dependencies.